app/main.py

```diff
@@ -1,8 +1,56 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int
+                 ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(
+            (car.comfort_class * (self.clean_power - car.clean_mark)
+             * (self.average_rating / self.distance_from_city_center)), 1)
+
+    def wash_single_car(self, car: Car) -> float:
+        if car.clean_mark > self.clean_power:
+            return 0
+        income = 0
+        income += self.calculate_washing_price(car)
+        car.clean_mark = self.clean_power
+        return income
+
+    def rate_service(self, rate: int) -> None:
+        old_count_of_ratings = self.count_of_ratings
+        self.count_of_ratings += 1
+        self.average_rating = round(
+            (self.average_rating * old_count_of_ratings + rate)
+            / self.count_of_ratings,
+            1)
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+        for car in cars:
+            income += self.wash_single_car(car)
+        return income
+
+
+def serve_cars(cars: list) -> float:
+    income = 0
+    for car in cars:
+        income += CarWashStation.wash_single_car(car)
+    return income
```

Comment:
The `serve_cars` function is attempting to call `wash_single_car` as if it were a class method of `CarWashStation`, which is incorrect. `wash_single_car` is an instance method and should be called on an instance of `CarWashStation`. You need to create an instance of `CarWashStation` and call `wash_single_car` on that instance for each car.